### ISSUE TITLE
Conflict with symfony 2.3.*

### ISSUE THREAD

**Issue by Chrysweel** [symfony-cmf/BlockBundle#240]
Hello. I must update composer but today I get conflict.

```
....continue...
 Conclusion: don't install symfony/symfony v2.3.12
    - Conclusion: don't install symfony/symfony v2.3.11
    - Conclusion: don't install symfony/symfony v2.3.10
    - Conclusion: don't install symfony/symfony v2.3.9
    - Conclusion: don't install symfony/symfony v2.3.8
    - Conclusion: don't install symfony/symfony v2.3.7
    - Conclusion: don't install symfony/symfony v2.3.6
    - Conclusion: don't install symfony/symfony v2.3.5
    - Conclusion: don't install symfony/symfony v2.3.4
    - Installation request for craue/formflow-bundle dev-master -> satisfiable by craue/formflow-bundle[dev-master].
    - Conclusion: don't install symfony/symfony v2.3.3
    - Installation request for sylius/sylius 0.10.0 -> satisfiable by sylius/sylius[v0.10.0].
    - Conclusion: don't install symfony/symfony v2.3.2
    - Conclusion: don't install symfony/symfony v2.3.1
    - sylius/sylius v0.10.0 requires symfony-cmf/block-bundle 1.1.* -> satisfiable by symfony-cmf/block-bundle[1.1.0, 1.1.0-RC1, 1.1.0-RC2, 1.1.0-RC3, 1.1.1, 1.1.2, 1.1.x-dev].
    - symfony-cmf/block-bundle 1.1.0 conflicts with symfony/symfony[v2.3.0].
    - symfony-cmf/block-bundle 1.1.0-RC1 conflicts with symfony/symfony[v2.3.0].
    - symfony-cmf/block-bundle 1.1.0-RC2 conflicts with symfony/symfony[v2.3.0].
    - symfony-cmf/block-bundle 1.1.0-RC3 conflicts with symfony/symfony[v2.3.0].
    - symfony-cmf/block-bundle 1.1.1 conflicts with symfony/symfony[v2.3.0].
    - symfony-cmf/block-bundle 1.1.2 conflicts with symfony/symfony[v2.3.0].
    - symfony-cmf/block-bundle 1.1.x-dev conflicts with symfony/symfony[v2.3.0].
....continue...
```

Any idea ? Can I continue use the block bundle with symfony 2.3.* version ?

It is my composer.json
```
        "php": ">=5.3.3",
        "symfony/symfony": "2.3.*",
        "doctrine/orm": "~2.2,>=2.2.3",
        "doctrine/doctrine-bundle": "~1.3@dev",
        "twig/extensions": "1.0.*",
        "symfony/assetic-bundle": "2.3.*",
        "symfony/swiftmailer-bundle": "2.3.*",
        "symfony/monolog-bundle": "2.3.*",
        "sensio/distribution-bundle": "2.3.*",
        "sensio/framework-extra-bundle": "2.3.*",
        "sensio/generator-bundle": "2.3.*",
        "incenteev/composer-parameter-handler": "~2.0",
        
        "stof/doctrine-extensions-bundle": "1.1.*@dev",
        
        "sonata-project/admin-bundle": "2.3.*@dev",
        "sonata-project/core-bundle": "~2.2",
        "sonata-project/doctrine-orm-admin-bundle": "2.3.*@dev",
        "sonata-project/doctrine-extensions": "1.0.2",
        
        "mopa/bootstrap-bundle": "dev-master",
        "twbs/bootstrap": "dev-master",
        "knplabs/knp-paginator-bundle": "dev-master",
        "knplabs/knp-menu": "1.1.*",
        "craue/formflow-bundle": "dev-master",        
        
        "shtumi/useful-bundle": "dev-master",
        "sonata-project/formatter-bundle": "2.3.2",
	"frequence-web/contact-bundle": "*",
        "friendsofsymfony/rest-bundle": "1.3.*",
        "jms/serializer-bundle": "0.12.0",
        "sylius/sylius": "0.10.0",
        "presta/sitemap-bundle": "~1.3@dev"
```

Thanks by advance !

**Comment by lsmith77** [comment]
which bundle has the dependency on CmfBlockBundle?

**Comment by Chrysweel** [comment]
Hello @lsmith77 thanks for to reply !

Sylius require this bundle: 
```
    "name": "sylius/sylius",
    "version": "v0.10.0",
```

**Comment by lsmith77** [comment]
I think something else might be going on ..
https://packagist.org/packages/sylius/sylius#v0.10.0 depends on CmfBlockBundle 1.1
https://packagist.org/packages/symfony-cmf/block-bundle#1.1.0 allows Symfony 2.3 (in fact all of our released versions of all CMF packages allowed Symfony 2.3)

I think your issue might be https://packagist.org/packages/craue/formflow-bundle instead

**Comment by Chrysweel** [comment]
Exactly. 

I fixed version of formflow-bundle to 2.1.8 and all correct ! 

Thanks @lsmith77 !
